### PR TITLE
Ranged seelie nerf

### PR DESCRIPTION
**Pull request by wow50000**
## About The Pull Request
Reduces seelie ranged damage to be in line with what their melee damage should be (10% due to 1 str)

## Why It's Good For The Game
Seelies are not supposed to engage in combat and if they are they are meant to do very little damage, nerfing the ranged damage nerfs the crit rate for seelies using bows and crossbows in line with what their melee damage is.

### CODE REVIEW

**Comment by Nakalon** [comment]
I agree with this. I think seelies are and should be mages by definition!

**Comment by Simply-Sydney** [comment]
Seelie are not meant to engage in combat as their primary means of interaction that is true.

However, it is an unfortunate fact that they are treated by most players as an annoying nuisance and killed with little reason other than 'lol I hate them'.

They need a way to defend themselves - I have seen half a dozen players tell me they stopped playing Seelie, and some that stopped playing entirely, for this issue alone.

Powergaming and bad faith play should be dealt with, but not through mechanic changes that punish everyone unilaterally and make the role even more frag bait than it already is.

**Comment by Simply-Sydney** [comment]
TL;DR - salt PR, please dont

**Comment by Nakalon** [comment]
> Seelie are not meant to engage in combat as their primary means of interaction that is true.
> 
> However, it is an unfortunate fact that they are treated by most players as an annoying nuisance and killed with little reason other than 'lol I hate them'.
> 
> They need a way to defend themselves - I have seen half a dozen players tell me they stopped playing Seelie, and some that stopped playing entirely, for this issue alone.
> 
> Powergaming and bad faith play should be dealt with, but not through mechanic changes that punish everyone unilaterally and make the role even more frag bait than it already is.

Thats a player problem, which needs to be adressed, making Seelies more common place would help solve this in my view

**Comment by Purpleturtle3** [comment]
Uhm, why are you putting those templar changes in here?

**Comment by Nakalon** [comment]
Although I'm generally for this, this is a balance change and I'd rather have staffs opinion on this!

**Comment by wow50000** [comment]
> Uhm, why are you putting those templar changes in here?

my dumb ass made some changes to main, reverted them
